app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,94 @@
+require './config/environment'
+
+class TweetsController < Sinatra::Base
+  include Helper
+
+  configure do
+    enable :sessions unless test?
+    set :public_folder, 'public'
+    set :views, 'app/views'
+    set :session_secret, "secret"
+  end
+
+  get '/tweets' do
+    if !is_logged_in
+      # redirect to '/signup'
+      redirect to '/login'
+    else
+      @user = current_user
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    end
+  end
+
+  get '/tweets/new' do
+    if !is_logged_in
+      redirect to '/login'
+    else
+      @user = current_user
+      erb :'tweets/create_tweet'
+    end
+  end
+
+  post '/tweets' do
+    if !is_logged_in
+      redirect to '/login'
+    else
+      @tweets = Tweet.create(user_id: current_user.id, content: params[:content])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.